view/adminhtml/templates/order/view/tab/info/payment.phtml

```diff
@@ -23,15 +23,29 @@ if (!$block->isBoltOrder($_order)) return;
 <div class="admin__page-section-item-content">
     <table class="admin__table-secondary order-information-table">
         <tbody>
-        <tr>
-            <th><?php echo __('Credit Card Type') ?></th>
-            <td><?php echo strtoupper($_order->getPayment()->getCcType()); ?></td>
-        </tr>
-        <tr>
-            <th><?php echo __('Credit Card Numbe') ?></th>
-            <td><?php echo sprintf('xxxx-%s', $_order->getPayment()->getCcLast4()); ?>
-            </td>
-        </tr>
+		<?php
+		$bolt_processor = $_order->getPayment()->getAdditionalInformation('processor');
+		if ( $bolt_processor == 'vantiv' ) {
```

Comment:
Perhaps the processor should not be hard coded here. You could take it out into a constant and make it an array (of one item for now). Sooner or later we will support more processors. Just a recommendation. Other than that LGTM. Approving.

Reply:
@danac-gs I have updated the PR per your comment, please review again